cns/configuration/configuration.go

```diff
@@ -17,6 +18,8 @@ const (
 
 type CNSConfig struct {
 	TelemetrySettings TelemetrySettings
+	ManagedSettings   ManagedSettings
+	ChannelMode       string
```

Comment:
is ChannelMode supposed to be specified in config file? I don't see it in the config above.

Reply:
if you're referring to cns_config.json then I just added it